platform/src/test/resources/test_assets_duplicate_asset.json

```diff
@@ -0,0 +1,168 @@
+{
+  "assets": [
```

Comment:
To simplify and focus the test, can we use minimal JSON file. 

For example, remove the `deposit`, `withdraw`, `send`, `sep31`, `sep38` fields?